src/applications/disability-benefits/526EZ/components/SelectArrayItemsWidget.jsx

```diff
@@ -19,10 +19,16 @@ export default class SelectArrayItemsWidget extends React.Component {
       value: items,
       id,
       options,
-      required
+      required,
+      formContext
     } = this.props;
     const { label: Label, selectedPropName, disabled } = options;
 
+    if (formContext.reviewMode) {
+      return (
+        <ul>{items.filter(i => i['view:selected']).map((d, i) => <li key={i}>{d.name}</li>)}</ul>
```

Comment:
Easier to read if this uses descriptive names like `item` or `data`. also easier to read if the 
`.dotFunctions()  // because you can comment`
`.areOnDifferentLines() // on each one`

Reply:
Thanks @rianfowler, great points, changed.